example/provider-local/base/controller-registration.yaml

```diff
@@ -19,6 +19,10 @@ spec:
     deploymentRefs:
     - name: provider-local
   resources:
+  - kind: BackupBucket
+    type: Local
```

Comment:
```suggestion
    type: local
```

Reply:
Some changes in the `Etcd-druid` and probably to the `backup-restore` would be needed to support that
https://github.com/gardener/gardener/pull/5514#discussion_r820425062